### DISCUSSION TITLE
More than 15,000 'grammar' tokens have been presented. To prevent Denial Of Service attacks, parsing has been cancelled.

### DISCUSSION THREAD

**Discussion by 1450793561** [apache/skywalking#11672] [Q&A]
Open Topology report error.I don't see the configuration on the oap service
How i resolve this?

<img width="1394" alt="企业微信截图_0fde292d-7a46-414a-b594-33d2130d297a" src="https://github.com/apache/skywalking/assets/31468018/2c1fc011-e2fd-4a64-a1c7-9c947d26f855">

**Comment by wu-sheng** [comment]
Do you have super large number of services? It is better you could provide your GraphQL(HTTP request) for this specific query?
You should be able to find it in the Browser.

**Reply by 1450793561** [reply]
Yes, there are many services.
I don't quite understand what you mean

**Reply by wu-sheng** [reply]
This is a error due to service side rejection. If you are a developer or ask for help from a developer, he/she will know how to get this specific HTTP request.

**Reply by 1450793561** [reply]
I'm an operations support person, not a developer. Specific HTTP request What is this? Is there a manual?

**Reply by wu-sheng** [reply]
No. We don't have that. Ask for your dev team to help.
